src/BloomBrowserUI/publish/PDFPrintPublish/PDFPrintFeaturesGroup.tsx

```diff
@@ -17,7 +17,7 @@ import { kSelectCss } from "../../bloomMaterialUITheme";
 import { BloomTooltip } from "../../react_components/BloomToolTip";
 import { ToggleButton, Typography } from "@mui/material";
 
-interface PdfReadyMessage {
+export interface PdfReadyMessage {
```

Comment:
I use this it in publish tab, to be able to check if a PDF has been generated so we will know whether to include Download PDF options in the context menu